stylish/index.js

```diff
@@ -26,17 +45,18 @@ module.exports = {
         'newlines-between': 'always',
       },
     ],
-    'no-else-return': 'error',
-    'no-extra-bind': 'error',
-    'no-extra-label': 'error',
-    'no-floating-decimal': 'error',
-    'no-multi-spaces': 'error',
+
+    '@stylistic/arrow-parens': ['error', 'always'],
+    '@stylistic/arrow-spacing': 'error',
+    '@stylistic/block-spacing': 'error',
+    '@stylistic/dot-location': ['error', 'property'],
+    '@stylistic/no-floating-decimal': 'error',
+    '@stylistic/no-multi-spaces': 'error',
     'prefer-arrow-callback': 'error',
-    'prefer-destructuring': 'error',
```

Comment:
missing?

Reply:
This one I intentionally disabled because in practice it produces more harm than good.

One clear example is that it forces you to change:

```typescript
let thing

if (otherThing.length) {
  thing = otherThing[0]
}
```

To this:

```typescript

let thing

if (otherThing.length) {
  ;[thing] = otherThing
}
```

Which is absurd.

The other thing is that destructing, even tho implemented in all engines almost a decade ago and optimized the hell of it, is still slower than direct property access. It will usually be negligible for most of the kind of stuff we write at Masterworks, but still one less reason to force it.